### PR TITLE
deps: uplift crd-ref-docs to 0.1.0

### PR DESCRIPTION
**Pull request by zdtsw**
## Description

- their release changed and support different os/arch binary
- and it is named as x86_64 not amd64 :shushing_face: 

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -264,12 +264,23 @@ golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
 $(GOLANGCI_LINT): $(LOCALBIN)
 	test -s $(GOLANGCI_LINT) || { curl -sSfL $(GOLANGCI_LINT_INSTALL_SCRIPT) | bash -s $(GOLANGCI_LINT_VERSION); }
 
-CRD_REF_DOCS_DL_URL ?= 'https://github.com/elastic/crd-ref-docs/releases/download/v$(CRD_REF_DOCS_VERSION)/crd-ref-docs'
+
+ifeq ($(shell go env GOOS), linux)
+ARCH=x86_64
+else
+ARCH=$(strip $(shell go env GOARCH))
+endif
```
Comment (ykaliuta) [inline Makefile:272]: Their OS and ARCH values are `uname -s` and `uname -m`

File: Makefile [hunk 2]
```diff
@@ -264,12 +264,23 @@ golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
...
+CRD_REF_DOCS_TARBALL="crd-ref-docs.tar.gz"
 .PHONY: crd-ref-docs
-crd-ref-docs: $(CRD_REF_DOCS) ## Download crd-ref-docs locally if necessary.
+crd-ref-docs: $(CRD_REF_DOCS)
 $(CRD_REF_DOCS): $(LOCALBIN)
-	test -s $(CRD_REF_DOCS) || curl -sSLo $(CRD_REF_DOCS) $(CRD_REF_DOCS_DL_URL) && \
-	chmod +x $(CRD_REF_DOCS) ;\
+	test -s $(CRD_REF_DOCS) || ( \
+		OS=$(shell go env GOOS) && \
```
Comment (ykaliuta) [inline Makefile:278]: why not make variable?

File: Makefile [hunk 3]
```diff
@@ -264,12 +264,23 @@ golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
...
+		wget -q -O $(LOCALBIN)/$(CRD_REF_DOCS_TARBALL) https://github.com/elastic/crd-ref-docs/releases/download/v$(CRD_REF_DOCS_VERSION)/crd-ref-docs_$(CRD_REF_DOCS_VERSION)_$${OS}_${ARCH}.tar.gz && \
+		tar -xzf $(LOCALBIN)/$(CRD_REF_DOCS_TARBALL) -C $(LOCALBIN) && \
+		rm -rf $(LOCALBIN)/$(CRD_REF_DOCS_TARBALL) $(LOCALBIN)/LICENSE $(LOCALBIN)/README.md && \
```
Comment (ykaliuta) [inline Makefile:281]: I would use something like
`curl -sSL <tarball url> | tar -zxf - -C $(LOCALBIN) crd-ref-docs`

File: Makefile [hunk 4]
```diff
@@ -264,12 +264,23 @@ golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
...
+CRD_REF_DOCS_TARBALL="crd-ref-docs.tar.gz"
 .PHONY: crd-ref-docs
-crd-ref-docs: $(CRD_REF_DOCS) ## Download crd-ref-docs locally if necessary.
+crd-ref-docs: $(CRD_REF_DOCS)
 $(CRD_REF_DOCS): $(LOCALBIN)
-	test -s $(CRD_REF_DOCS) || curl -sSLo $(CRD_REF_DOCS) $(CRD_REF_DOCS_DL_URL) && \
-	chmod +x $(CRD_REF_DOCS) ;\
+	test -s $(CRD_REF_DOCS) || ( \
+		OS=$(shell go env GOOS) && \
+		wget -q -O $(LOCALBIN)/$(CRD_REF_DOCS_TARBALL) https://github.com/elastic/crd-ref-docs/releases/download/v$(CRD_REF_DOCS_VERSION)/crd-ref-docs_$(CRD_REF_DOCS_VERSION)_$${OS}_${ARCH}.tar.gz && \
```
Comment (ykaliuta) [inline Makefile:279]: `$()` for make variables please

**Comment by ykaliuta** [comment]
amd64 is a legacy name from the times when Intel did not support it and promoted their VLIW IA64. Debian keeps it the same for the same reasons. But nowadays name is x86_64.

File: Makefile (hunk 1 above)
Comment (zdtsw) [reply Makefile]: i rewrite this whole shabnag

**Review by ykaliuta — APPROVED** [review]

**Pull request merged**